packages/shared/src/utils/helpers/validation-rules.ts

```diff
@@ -11,14 +18,14 @@ export const getContractValidationRules = () => ({
             [
                 'req',
                 {
-                    condition: contract_store => !contract_store.contract_update_stop_loss,
+                    condition: (contract_store: TContractStore) => !contract_store.contract_update_stop_loss,
                     message: localize('Please enter a stop loss amount.'),
                 },
             ],
             [
                 'custom',
                 {
-                    func: (value, options, contract_store) => {
+                    func: (value: number, contract_store: TContractStore) => {
```

Comment:
same for line 38 and line 62:
**options** is the 2nd parameter in this function, and `contract_store` is the 3rd.
If you change the signature of these functions, you have to also change their usage in `validator.js` files in various packages where we pass `options` into them as a 2nd parameter and `contract_store` - as the 3rd parameter.

Reply:
I added this! but we don't use it

Reply:
yes, correct, What I mean is in case we want to discard this parameter here, we have to update validator.js files where we pass these params in this order.  🙂  Or it's also good to leave it like it is, thank you 🙏🏻